Microsoft.DotNet.Interactive.PowerShell/PowerShellKernel.cs

```diff
@@ -71,6 +71,9 @@ public PowerShellKernel() : base(DefaultKernelName)
         {
             _psHost = new PSKernelHost();
             _lazyPwsh = new Lazy<PowerShell>(CreatePowerShell);
+
+            var command = new SubmitCode(DollarProfileHelper.GetProfileScript());
```

Comment:
I suggest to have a `PowerShellKernelExtensions.cs` just like c#/f# kernels, and have an extension method called `UseProfiles` which adds 1 or 2 `SubmitCode` as needed (one for `alluserscurrenthost`, one for `currentusercurrenthost`).

Reply:
Good call - I've added that.